modules/cms/twig/ComponentNode.php

```diff
@@ -34,7 +34,7 @@ public function compile(Twig_Compiler $compiler)
         }
 
         $compiler
-            ->write("echo \$this->env->getExtension('Cms\Twig\Extension')->componentFunction(")
+            ->write("echo \$this->env->getExtension('Cms\Twig\Extension')->componentFunction(\$context,")
```

Comment:
What's the initial backslash on $context for?

Reply:
Escapes the $ in the double-quoted string, otherwise PHP tries to do variable expansion.

Reply:
I gotcha. Wouldn't it be simpler to reverse the order of the single and double quotes then? Or am I missing something else?

Reply:
I just added the $context variable to what was there already. If you swap them wouldn't you then need to double escape the backslashes anyway?

Reply:
Fair point, that would be the case.